viewing_party/party.py

```diff
@@ -1,23 +1,138 @@
+from re import T
+
 # ------------- WAVE 1 --------------------
 
+# This function adds 3 key-value pairs to a dictionary called movie_dict
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict

```

Comment:
I think it's clearer to directly return a dictionary literal instead of creating and modifying an empty one:

```suggestion
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
```